scripts/docker/_RosTeamWs_Docker_Defines.bash

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022, Stogl Robotics Consulting UG (haftungsbeschränkt)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Load Framework defines
+docker_script_own_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+source $docker_script_own_dir/../_RosTeamWs_Defines.bash
+
+# assoziative array which maps the chosen rosdistro to the name which is used to install the corresponding
+# ros distribution inside docker
+declare -r -A map_to_docker_ros_distro_name=( ["foxy"]="foxy" ["galactic"]="galactic" ["rolling"]="rolling" )
+declare -r -A ros_distro_to_rtw_branch=( ["foxy"]="foxy" ["galactic"]="master" ["rolling"]="master")
+
+# $1 = name (tag) of the created docker image
+# $2 = Dockerfile which is used for creating the image
+build_docker_container () {
```

Comment:
Looking at these methods, do we really need `recreate_docker.sh` in each docker folder?

I don't see much point of it because we are anyway using inputs as here too. I am probably missing something. You are welcome to correct me.

Reply:
Going to merge this, was just the quickest way